dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -643,6 +643,11 @@ protected void handlePublishUnpublish(String entryPath, String newName, boolean
                 out("This workflow is already unpublished.");
             }
         } else {
+            if (newName != null) {
+                errorMessage("Parameter '--entryname' currently only supported for tools.", CLIENT_ERROR);
+            }
+            /*
```

Comment:
Commented this out since its not working, but might look into it later. Could delete it and look up previous version instead.

Reply:
I could see it both ways, but since you have workflow specific help, I would probably just not list the option at all, rather than listing it and saying it doesn't work. Curious what others think.

Reply:
I don't have it listed as an option in workflow help, but I included the error here in case someone actually uses it. This method is overridden and the original abstract entry method takes new name as a parameter.

Reply:
I'm ok with simplifying and taking out the option entirely if it never worked.